transport_and_format.asciidoc

```diff
@@ -312,81 +312,83 @@ include::examples/transport_and_format_get_token_list_example.json[]
+In other words: the Hub broadcasts received information to all connected clients.
 
-To send a Broadcast push, the client uses to Hubs party-id and country-code in the 'OCPI-to-' headers.
-The Hub parses the request and sends a response to the client, with optionally its own party-id and country-code in the 'OCPI-from-' headers.
-The Hub then sends the push to any client implementing the applicable module, using its own party-id and country-code in the 'OCPI-from-' headers.
-The client receiving a push from a Hub (with the Hubs information in the 'OCPI-from-' headers) will respond to
-this push with the Hubs party-id and country-code in the 'OCPI-to-' headers.
+To send a broadcast push, the client uses the party-id and country-code of the Hub in the 'OCPI-to-' header.
+The Hub parses the request and sends a response to the client, which optionally contains its own party-id and country-code in the 'OCPI-from-' header.
+The Hub then sends the pushed data to any client implementing the corresponding applicable module, using its own party-id and country-code in the 'OCPI-from-' header.
+The client receiving a PUSH from a Hub (with the Hubs information in the 'OCPI-from-' header) will respond to
+this PUSH with the Hubs party-id and country-code in the 'OCPI-to-' header.
 
-GET SHALL NOT be used in combination with Broadcast push.
+GET SHALL NOT be used in combination with broadcast pushing.
 If the requesting party wants to GET information of which it does not know the receiving party,
-to <<transport_and_format_message_routing_open_routing_request,Open routing request>> MUST be used. (see below)
+an <<transport_and_format_message_routing_open_routing_request,open routing request>> MUST be used. (see below)
 
-Broadcast push SHALL only be used with information that is meant to be send to all other parties. It is useful for things like:
+Broadcast pushing SHALL only be used with information that is meant to be sent to all other parties. It is useful to share data like
 <<mod_tokens.asciidoc#mod_tokens_tokens_module,Tokens>> and <<mod_locations.asciidoc#mod_locations_locations_module,Locations>>,
-but not for <<mod_cdrs.asciidoc#mod_cdrs_cdrs_module,CDRs>> and <<mod_sessions.asciidoc#mod_sessions_sessions_module,Sessions>>.
+but not so much for <<mod_cdrs.asciidoc#mod_cdrs_cdrs_module,CDRs>> and <<mod_sessions.asciidoc#mod_sessions_sessions_module,Sessions>>
+as these pieces of information are specific to only one party and are possibly even protected by GDPR or other laws.
 
-NOTE: For "Client owned objects" the party-id and country-code in the URL segments will still be the original party-id and country-code from the original client sending the Broadcast push to the Hub.
+NOTE: For "client owned objects", the party-id and country-code in the URL segments will still be the original party-id and country-code from the original client sending the Broadcast push to the Hub.
```

Comment:
Also `Client Owned Objects` is a sort of term used in OCPI. So maybe starting with capitals is better.

Reply:
What about the chapter heading `3.1.5. Client owned object push`? The `push` also capitalized?